dataherald/sql_database/base.py

```diff
@@ -88,13 +89,38 @@ def from_uri_ssh(cls, database_info: DatabaseConnection):
             f"{ssh.db_driver}://{ssh.remote_db_name}:{fernet_encrypt.decrypt(ssh.remote_db_password)}@{local_host}:{local_port}/{ssh.db_name}"
         )
 
+    @classmethod
+    def parser_to_filter_commands(cls, command: str) -> str:
+        sensitive_keywords = [
+            "DROP",
+            "DELETE",
+            "UPDATE",
+            "INSERT",
+            "GRANT",
+            "REVOKE",
+            "ALTER",
+            "TRUNCATE",
+            "MERGE",
+            "EXECUTE",
+        ]
+        pattern = (
+            r"\b(?:" + "|".join(re.escape(word) for word in sensitive_keywords) + r")\b"
+        )
+        match = re.search(pattern, command, re.IGNORECASE)
+        if match:
+            raise Exception(
```

Comment:
Instead of raising an `Exception` we should create our own `Exception` and raise that one for example:

`
class SensitiveSQLException(Exception):
    """Raised when a sql has a sensitive word"""
    pass
` 
then you can do:
`
raise SensitiveSQLException(.......)
`
And catch in the controller this exception to return a 400 error